src/baldr/graphreader.cc

```diff
@@ -157,24 +280,40 @@ TileCache* TileCacheFactory::createTileCache(const boost::property_tree::ptree&
 
   size_t max_cache_size = pt.get<size_t>("max_cache_size", DEFAULT_MAX_CACHE_SIZE);
 
+  bool use_lru_cache = pt.get<bool>("use_lru_mem_cache", false);
+  auto lru_mem_control = pt.get<bool>("lru_mem_cache_hard_control", false)
```

Comment:
@dnesbitt61 these are the config options. you can see the default is to use the old tile cache style, where `trim==clear`